oaknational/Oak-Web-Application
# Fix/pupil 613/404 reloading

## Description
Currently we overwrite the url to add a path for section when it's not provided. When the lesson isn't found we go to the 404 . However that causes the overwrite action to happen again and we land back at the original page and rinse and repeat

Music year: 1977

- changed rewrites to redirects
- these are permanent redirects

## How to test

1. Go to https://deploy-preview-2364--oak-web-application.netlify.thenational.academy/pupils/lessons/transverse-waves
See that the page loads as expected and /overview is appended to the url
2. Go to https://deploy-preview-2364--oak-web-application.netlify.thenational.academy/pupils/lessons/transverse-waves-bad
See that the 404 loads but without perpetual reloading


## ACS

[x] page no longer perpetually reloads
